web/docs/guides/database/extensions/dblink.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: dblink
+title: 'dblink: Connect Databases'
+description: Connect to other PostgreSQL databases from within a database.
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+The `dblink` extension allows you to connect to other PostgreSQL databases from within a database.
+
+## Overview
+
+The `dblink` is an extension which allows to connect to other PostgreSQL database within a database. With `Foreign Data Wrappers (FDW)` you can uniquely define a remote foreign server in order to access its data.
+
+## Usage
+
+`dblink` is used To connect to other PostgreSQL databases from within a database.
+
+### Enabling
+
+<Tabs
+defaultValue="UI"
+values={[
+  {label: 'UI', value: 'UI'},
+  {label: 'SQL', value: 'SQL'},
+]}>
+<TabItem value="UI">
+
+```sh
+1. Go to the Database page.
+2. Click on "Extensions" in the sidebar.
+3. Search for "dblink".
+4. Click the toggle.
+```
+
+</TabItem>
+<TabItem value="SQL">
+
+```sql
+
+--Example: enable the "dblink" extension and ensure it is installed
+CREATE EXTENSION dblink SCHEMA extensions;
+
+```
+
+</TabItem>
+
+</Tabs>
+
+### Test connection of database
+
+```
+SELECT dblink_connect('host=localhost user=postgres password=enjoy dbname=postgres');
```

Comment:
in this case I don't know if we should use `localhost`, because the database will never know how to find it. Perhaps we use a cloud host name (like another supabase database?)

Reply:
@kiwicopple What do you think of the database name?